tchannel/tchannel.py

```diff
@@ -206,7 +206,7 @@ def decorator(fn):
             return decorator(handler)
 
     @gen.coroutine
-    def advertise(self, routers, name=None, timeout=None):
+    def advertise(self, routers, name=None, timeout=None, router_file=None):
```

Comment:
Need to explain what the router_file's format. Other users can put any format content in the file.